clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -375,3 +376,30 @@ class KubernetesSeLinuxOptions {
   String type
   String level
 }
+
+@AutoClone
+@Canonical
+//Base on model http://www.javadoc.io/doc/io.fabric8/kubernetes-model/1.0.73
```

Comment:
I would try to follow the model spec here https://godoc.org/k8s.io/kubernetes/pkg/api#PodSpec rather than the Fabric8 pod spec, since the former is ultimately what we want to model